packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -393,7 +393,7 @@ class DropdownMenu<T> extends StatefulWidget {
   /// properties are used.
   ///
   /// Defaults to null.
-  final EdgeInsets? expandedInsets;
+  final EdgeInsetsGeometry? expandedInsets;
```

Comment:
This is going to need a test. What if you pump a `DropdownMenu` in a widget test and pass an `EdgeInsetsDirectional` to it? Then assert that the padding is the EdgeInsetsDirectional that you passed in.

Reply:
Done